third_party/gpus/cuda_configure.bzl

```diff
@@ -50,6 +50,7 @@ load(
     "raw_exec",
     "read_dir",
     "which",
+    "realpath",
```

Comment:
This should be sorted in alphabetical order for linter and code-style check.

Reply:
Resolved.